server/tidb_test.go

```diff
@@ -130,9 +130,9 @@ func createTidbTestTopSQLSuite(t *testing.T) (*tidbTestTopSQLSuite, func()) {
 	}()
 
 	dbt := testkit.NewDBTestKit(t, db)
-	dbt.MustExec("set @@global.tidb_top_sql_precision_seconds=1;")
-	dbt.MustExec("set @@global.tidb_top_sql_report_interval_seconds=2;")
-	dbt.MustExec("set @@global.tidb_top_sql_max_statement_count=5;")
+	topsqlstate.GlobalState.PrecisionSeconds.Store(1)
+	topsqlstate.GlobalState.ReportIntervalSeconds.Store(2)
```

Comment:
These two tests permently modifies the topsqlstate.GlobalState. Will it affect and pollute other tests?

Reply:
All topsql related tests in this pkg will use this to setup,  so it's ok here?

Reply:
ok, already add restore default logic.